src/helpers/addError.ts

```diff
@@ -0,0 +1,16 @@
+import { FormContext } from "../Form";
+import { ModelError } from "../Model";
+
+export function addError(context: FormContext, error: any): void {
+    if (!(error.response && error.response.status === 400)) {
+        throw error;
+    }
+
+    error.response.data.errors.forEach(context.addError);
```

Comment:
Will fail if data is not object or have no errors property.
Each error should be validated. 
Because react-context-form is not our internal product we can't know how it will be used.